services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
```

Comment:
Text content: There is variation in the image caption font size and missing bolded words that appear in CDS.